src/main/java/com/emc/pravega/perf/PravegaReaderWorker.java

```diff
@@ -30,60 +28,48 @@
 import io.pravega.client.stream.EventRead;
 import io.pravega.client.stream.ReinitializationRequiredException;
 
-public class PravegaReaderWorker implements Callable<Void> {
-    public final static AtomicInteger eventCount = new AtomicInteger(0);
-    public final long totalEvents;
+
+/**
+ *  class for Pravega reader/consumer.
+ */
+public class PravegaReaderWorker extends ReaderWorker  {
+    private final static AtomicInteger eventCount = new AtomicInteger(0);
```

Comment:
Making this static means that two instances of this test can never be run in the the same VM. That's a really odd restriction to impose, especially without documentation. Wouldn't a target rate an duration be sufficient to control when to exit?

Reply:
yes, its removed now